modules/react/_examples/stories/examples/CookieBanner.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {colors, commonColors, depth, type, space} from '@workday/canvas-kit-react/tokens';
+import {PrimaryButton, TertiaryButton} from '@workday/canvas-kit-react/button';
+
+const CookieBannerItem = createComponent('div')({
+  displayName: 'CookieBanner.Item',
+  Component: ({isRow, ...elProps}: ItemProps, ref) => (
+    <BannerItem ref={ref} isRow={isRow} {...elProps} />
+  ),
+});
+
+const CookieBanner = createComponent('div')({
+  displayName: 'CookieBanner',
+  Component: (props: CookieBannerProps, ref, Element) => (
+    <Banner ref={ref} as={Element} {...props} />
+  ),
+  subComponents: {Item: CookieBannerItem},
+});
+
+export const Basic = () => {
+  const DefaultNotice = `We use cookies to ensure that we give you the best experience on our website. 
+    If you continue without changing your settings, we'll assume that you are willing to receive cookies.`;
+
+  return (
+    <ExampleContainer>
+      <CookieBanner isClosed={false}>
+        <CookieBanner.Item>{DefaultNotice}</CookieBanner.Item>
+        <CookieBanner.Item isRow>
+          <TertiaryButton>Settings</TertiaryButton>
+          <PrimaryButton>Continue</PrimaryButton>
+        </CookieBanner.Item>
+      </CookieBanner>
+    </ExampleContainer>
+  );
+};
+
+interface CookieBannerProps extends StyledType {
+  isClosed?: boolean;
+}
+
+interface ItemProps extends StyledType {
+  isRow?: boolean;
+}
```

Comment:
Little nitpicks. I don't think these 2 prop interfaces need to extend `StyledType`. `StyledType` adds the `as` prop to a styled component. These are used inside a `createComponent` which already adds the `as` prop correctly. Extending `StyledType` should be removed. This way, `Banner` doesn't have to use `Pick<CookieBannerProps, 'isClosed'>` since that is the only prop available.

```suggestion
interface CookieBannerProps {
  isClosed?: boolean;
}

interface ItemProps {
  isRow?: boolean;
}
```